### DISCUSSION TITLE
Can I show blogtags with the number for each tag?

### DISCUSSION THREAD

**Discussion by Zauberweib** [datenstrom/community#117] [Ask a question] [answered]
Is it possible to show in sidebar-list (blogtags) how often each tag is given? i.e.:
css (5)
format (7)
baking (3)
todo (16)
...
thanks,
zaubi

**Comment by markseuffert** [comment]
For writers there are [shortcuts](https://github.com/annaesvensson/yellow-blog#how-to-show-blog-information), for example use `[blogtags]` for a simple list of tags.
For developer there are more possibilities with customisation to make the website you want.

Hope this helps. :smile:

**Comment by Zauberweib** [comment]
not really :) I'm able to let count the tags in the list (=1) or let count all tags in the list (sets the same number to each tag; e.g. 4), but I didn't get it counting each tag itself and show in the list.

I'm looking in "php für Dummies" but I didn't get it anyway. I think I have to build an (new) array with values and names of tags.... html & css is easier than php... ;-)

**Comment by Zauberweib** [comment, marked as answer]
ha! I found it! _gg_ no big thing - but for me! :) 

Its only 
`$output .= htmlspecialchars($key)."</a>\n";`  
`$output .= htmlspecialchars($value)."</li>\n";`

instead of
`$output .= htmlspecialchars($key)."</a></li>\n";`

:)

this in the plugin-folder blog.php below from
`if($name=="blogtags" && $shortcut)`
(its roundabout line 117)

**Comment by markseuffert** [comment]
Good to hear :+1:

**Comment by BsNoSi** [comment]
Is a complete template available? I think this is something to document, because I believe, this will be interesting for some more people, using Yellow.

**Comment by Zauberweib** [comment]
This is only these two lines modified in the blog.php in the plugin-folder. No need for a complete template; this passage could be designed via css, cause it has its own class ".blogtags"
